.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: python
 
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
```

Comment:
As stated in #5 - I'd like to drop support for all versions below Python3.6 - so this means I do want to support Python 3.6. Maybe I should have made that clearer in the issue description.

Can you please add Python 3.6? Thank you.

Also, CI currently fails.

Travis has changed which interpreters are pre-installed on a machine. In order to test different Python versions, you have to use the `matrix` keyword. I was not aware of this until yesterday, when we faced the same problem in another repository.

Please update the above code similar to 
https://github.com/morepath/morepath_cerebral_todomvc/commit/c5b413728e05e92423c162534c32e6b39ee1cb0f

Thank you!

Reply:
Done